internal/backends/winit/wasm_input_helper.rs

```diff
@@ -108,12 +108,12 @@ impl WasmInputHelper {
                         for ch in data.chars() {
                             window_adapter
                                 .window()
-                                .dispatch_event(WindowEvent::KeyPressed { text: ch })
+                                .dispatch_event(WindowEvent::KeyPressed { text: ch.into() })
```

Comment:
Should that not be `WindowEvent::KeyPressed { text: data.into() }`  and get rid of the for loop?

Reply:
Hmm, that's true!